### PR TITLE
Abduction surgery fix

### PR DESCRIPTION
**Pull request by Tayyyyyyy**
~Fixes a new abduction surgery runtime on using hands to remove organs from #8817.~
Makes the extract organ and insert gland of abduction surgery always succeed, rather than fail silently.

### CODE REVIEW

**Comment by Fox-McCloud** [comment]
You've been beat.

**Comment by Tayyyyyyy** [comment]
I blame 10 minute compile times 😒 Gonna remove the runtime fix from this

**Comment by Tayyyyyyy** [comment]
Okey doke, duplicate fix removed

File: code/game/gamemodes/miniantags/abduction/abduction_surgery.dm [hunk 1]
```diff
@@ -49,6 +49,9 @@
 		to_chat(user, "<span class='warning'>You don't find anything in [target]'s [target_zone]!</span>")
 		return 1
 
+/datum/surgery_step/internal/extract_organ/fail_step(mob/user, mob/living/carbon/target, target_zone, obj/item/tool, datum/surgery/surgery)
+	return end_step(user, target, target_zone, tool, surgery)
```
Comment (Fox-McCloud) [inline code/game/gamemodes/miniantags/abduction/abduction_surgery.dm]: I don't really like this. You can intentionally fail just to make the step shorter.

Comment (Tayyyyyyy) [reply code/game/gamemodes/miniantags/abduction/abduction_surgery.dm]: Okay, I'll add a failure message instead

File: code/game/gamemodes/miniantags/abduction/abduction_surgery.dm [hunk 2]
```diff
@@ -49,6 +49,10 @@
 		to_chat(user, "<span class='warning'>You don't find anything in [target]'s [target_zone]!</span>")
 		return 1
 
+/datum/surgery_step/internal/extract_organ/fail_step(mob/user, mob/living/carbon/target, target_zone, obj/item/tool, datum/surgery/surgery)
+	user.visible_message("<span class='warning'>[user]'s hand slips, failing to extract anything!</span>", "<span class='warning'>Your hand slips, failing to extract anything!</span>")
+	return 0
```
Comment (Fox-McCloud) [inline code/game/gamemodes/miniantags/abduction/abduction_surgery.dm]: `return FALSE`

File: code/game/gamemodes/miniantags/abduction/abduction_surgery.dm [hunk 3]
```diff
@@ -65,6 +69,10 @@
 	gland.insert(target, 2)
 	return 1
 
+/datum/surgery_step/internal/gland_insert/fail_step(mob/user, mob/living/carbon/target, target_zone, obj/item/tool, datum/surgery/surgery)
+	user.visible_message("<span class='warning'>[user]'s hand slips, failing to insert the gland!</span>", "<span class='warning'>Your hand slips, failing to insert the gland!</span>")
+	return 0
```
Comment (Fox-McCloud) [inline code/game/gamemodes/miniantags/abduction/abduction_surgery.dm]: `return FALSE`

**Pull request merged**